src/bot.coffee

```diff
@@ -221,16 +229,14 @@ class SlackBot extends Adapter
       # If the reaction is to a message, then the item.channel property will contain a conversation ID
       # Otherwise reactions can be on files and file comments, which are "global" and aren't contained in a conversation
       user.room = event.item?.channel # when the item is not a message this will be undefined
+      # Convert item user into a Hubot user
+      item_user = @robot.brain.userForId event.item_user.id, event.item_user
```

Comment:
what happens when there is no `item_user` property? wouldn't `event.item_user.id` cause an error at runtime (cannot find property `id` on undefined)? this is a use case when the reaction is on a message that was created by an Incoming Webhook.